### PR TITLE
Standardize logo, text, alignment for plugin grids

### LINKED ISSUE #833: gridsome: for index page grid items, add strict `y` boundary between text and image
Following from feedback here: https://meltano.slack.com/archives/C043R4WV6TS/p1663869223504869

I think we should probably have a fixed 'y' boundary between image and text. This will keep text in a row all starting from the same y position. Also would allow us to ensure that the text label always fits (up to two lines of it anyway) in the grid space below the image.

Presumably, this may shrink some images which currently are using proportionally more height of the overall box.

We can also reduce the font size slightly if that helps reduce space pressure.

For contrast, here are two renderings with where logos have very different 'y' dimensionality:

image

### PR DESCRIPTION
**Pull request by cjohnhanson**
Closes #833

### CODE REVIEW

**Comment by rwfeather** [comment]
Plugin grid seems tricky to get right, now the multi-line names are overlapping logos:

Screen Shot 2022-09-22 at 3 20 04 PM

Feels like something has got to give with the varying logo and text sizes.

File: src/pages/Extractors.vue [hunk 1]
```diff
@@ -30,13 +30,11 @@
...
-                class="place-self-center object-scale-down"
+                class="place-self-center"
               />
             </div>
-            <div class="row-span-1 self-end object-scale-down">
-              <p
-                class="font-bold underline text-xs md:text-lg justify-self-center place-self-end object-scale-down text-clip"
-              >
+            <div class="grid content-end row-span-5 self-stretch h-full">
+              <p class="font-bold underline text-xs lg:text-md justify-self-center text-clip" e>
```
Comment (rwfeather) [inline src/pages/Extractors.vue:37]: Extra text "e" in these

**Review by rwfeather — APPROVED** [review]

**Comment by cjohnhanson** [comment]
> Plugin grid seems tricky to get right, now the multi-line names are overlapping logos:

@rwfeather yeah I think what we want here would be "make all cards the height of the the tallest content and put them in a 4-column grid". Sizing would be easier to do with flexbox but the grid obviously is easier to do with grid.

**Review by tayloramurphy — CHANGES_REQUESTED** [review]
@cjohnhanson anything we can do about that overlap? I'd be cool with assuming 2 lines of text and keeping logos to the max size of CSV (or whatever the largest one is).

Also, I just noticed that the alphabetization is case sensitive Screen Shot 2022-09-22 at 4 30 40 PM

**Comment by aaronsteers** [comment]
> I'd be cool with assuming 2 lines of text

Agreed. Let's assume 1 or 2 lines of text max.

**Comment by cjohnhanson** [comment]
@tayloramurphy @aaronsteers I took another pass at this this morning with hard-coded heights and it's looking better at the screen sizes I checked out. I also updated sorting to be case insensitive.

**Review by tayloramurphy — APPROVED** [review]
Love the changes! Looks great

**Review by aaronsteers — APPROVED** [review]
Looks really good! Thanks, @cjohnhanson !

**Pull request merged**